src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__calendar calendar calendar--month-length_29">
+      <div class="calendar__day day day--start-day_sun">
+      </div>
+      <div class="calendar__day day"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
Тут тебе должно хватить вот этого класса